src/pages/index.js

```diff
@@ -55,36 +60,39 @@ const IndexPage = ({ data }) => {
 
         <Numbers>
           {homepageQuery.numbers.map((object, i) => (
-
             <Number key={i}>
-            
               <div>
-                <NumberText>{object.number.split(" ").length > 1 ? <div>{object.number.split(" ")[0]} <span>{object.number.split(" ")[1]}</span></div> : object.number.split(" ")[0]}</NumberText>
+                <NumberText>
+                  {object.number.split(" ").length > 1 ? (
+                    <div>
+                      {object.number.split(" ")[0]}{" "}
+                      <span>{object.number.split(" ")[1]}</span>
+                    </div>
+                  ) : (
+                    object.number.split(" ")[0]
+                  )}
+                </NumberText>
                 <NumberDescription>
                   {object.number_description}
                 </NumberDescription>
               </div>
             </Number>
           ))}
-
-          {/* <Number>
-            <div>
-              <NumberText>{homepageQuery.numbers[1].number}</NumberText>
-              <NumberDescription>
-                {homepageQuery.numbers[1].number_description}
-              </NumberDescription>
-            </div>
-          </Number>
-          <Number>
-            <div>
-              <NumberText>{homepageQuery.numbers[2].number}</NumberText>
-              <NumberDescription>
-                {homepageQuery.numbers[2].number_description}
-              </NumberDescription>
-            </div>
-          </Number> */}
         </Numbers>
       </ByTheNumbers>
+
+      <SupporterImagesSection>
+        <SupporterImages>
+          {homepageQuery.supporter_images.map((image, id) => (
+            <SupporterImage alt="Supporter Logo" src={image.image.url} />
```

Comment:
i think it would be best to build into your content model somewhere that your client can add alt text so that the alt text can include names of companies (would be more meaningful to sight impaired users) - if this is already a field within the CMS you can pull it in from there or you can add another field